exercises/clock/clock.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Clock do
+  defstruct hr: 0, min: 0
+
+  @doc """
+  Returns a string representation of a clock:
+
+      iex> Clock.new(8, 9) |> to_s
+      "08:09"
+  """
+  @spec new(integer, integer) :: Clock
+  def new(hr, min) do
+  end
+
+  @doc """
+  Adds two clock times:
+
+      iex> Clock.add(10, 0, 3) |> to_s
+      "10:03"
+  """
+  @spec add(Clock, integer) :: Clock
+  def add(%Clock{hr: hr, min: min}, add_min) do
+  end
+
+  defimpl String.Chars, for: Clock do
```

Comment:
Hmm right. This does not give that error we were looking for. I guess I should remove the stub and go the rescue way giving explicit message to the user.